internal/provider/resource/secret_resource.go

```diff
@@ -85,6 +91,22 @@ func (r *secretResource) Schema(_ context.Context, _ resource.SchemaRequest, res
 				Optional:    true,
 				Description: "Tag ids to be attached for the secrets.",
 			},
+			"secret_reminder": schema.SingleNestedAttribute{
+				Attributes: map[string]schema.Attribute{
+					"note": schema.StringAttribute{
+						Description: "Note for the secret rotation reminder",
+						Computed:    false,
+						Optional:    true,
```

Comment:
The secret_reminder object is optional but the params of the secret_reminder should be required i think

Reply:
The note is optional, the repetition frequency isn't. On the UI we show it as a reminder to rotate the secret by default.
<img width="772" alt="Screenshot 2024-09-11 at 2 40 27 AM" src="https://github.com/user-attachments/assets/85b9bae9-3439-4780-beec-a7a95fe2d1f3">